File: app/src/main/java/eu/kanade/tachiyomi/ui/library/category/CategoryItemTouchHelper.java
package eu.kanade.tachiyomi.ui.library.category;

import eu.kanade.tachiyomi.ui.base.adapter.ItemTouchHelperAdapter;
import eu.kanade.tachiyomi.ui.base.adapter.SimpleItemTouchHelperCallback;

public class CategoryItemTouchHelper extends SimpleItemTouchHelperCallback {

    public CategoryItemTouchHelper(ItemTouchHelperAdapter adapter) {
        super(adapter);
    }

    @Override
    public boolean isItemViewSwipeEnabled() {
        return false;
    }
}

File: app/src/main/java/eu/kanade/tachiyomi/ui/category/CategoryItemTouchHelper.kt
package eu.kanade.tachiyomi.ui.category

import eu.kanade.tachiyomi.ui.base.adapter.ItemTouchHelperAdapter
import eu.kanade.tachiyomi.ui.base.adapter.SimpleItemTouchHelperCallback

class CategoryItemTouchHelper(adapter: ItemTouchHelperAdapter) : SimpleItemTouchHelperCallback(adapter) {

    /**
     * Disable items swipe remove
     *
     * @return false
     */
    override fun isItemViewSwipeEnabled(): Boolean {
        return false
    }

    /**
     * Disable long press item drag
     *
     * @return false
     */
    override fun isLongPressDragEnabled(): Boolean {
        return false
    }

}